changelog.md

```diff
@@ -1,5 +1,6 @@
 ### v4.5.6
 - Fix: hide edit pencils on iOS on H3, H4, H5 and H6
+- Fix: add a few css properties to read more part of the footer
```

Comment:
I know how we have this is a bit odd. These release notes are backwards facing the moment a release is cut and future facing before the next release. Since v4.5.6 has been published, we can't make changes to it. As this is a bug fix, please add a new v4.5.7 section. :+1: